dashboard/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:latest
+
+WORKDIR /app
+
+COPY requirements.txt .
+COPY streamlit_app.py .
+COPY .streamlit /.streamlit
+
+RUN pip install -r requirements.txt
+
+CMD streamlit run streamlit_app.py
```

Comment:
perfect! but maybe when you create the dockerfile have the RUN pip install after the requirements.txt so it doesnt have to run every time!